packages/http-client-java/generator/http-client-generator-standard-test/pom.xml

```diff
@@ -61,5 +61,23 @@
       <artifactId>slf4j-simple</artifactId>
       <version>1.7.36</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.18.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <version>5.11.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.54.1</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
We should not have azure-core here. Please delete it. Let us know if anything cannot be done without it and we need to find alternative.

Reply:
1. The authentication util class `AzureKeyCredentialPolicy` uses `ClientLogger`, which I think it can be removed
2. The test class `type.scalar.DecimalTests` uses `com.azure.core.annotation.Immutable`, and has not found alternative for it.

Reply:
1. Yeah, remove it. The class itself may not be necessary.
2. Just remove that `@Immutable`

Reply:
Fixed it by the [commit#f07eb05](https://github.com/weidongxu-microsoft/typespec/pull/1/commits/f07eb050c5e64d4829317ebde42e874935fc006a) and [commit#f385033](https://github.com/weidongxu-microsoft/typespec/pull/1/commits/f38503388fa1d664ef824edfe1e0110e58fdc455)